my_form.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Cam's Quiz</title>
+  <link rel="stylesheet" href="my_style.css">
+  <script src="nav.js"></script>
+  <script src="form_validate.js"></script>
+</head>
+
+<body onload="setNav(window.location.pathname)">
+  <nav id="main-nav"></nav>
+
+  <main>
+    <h1>What Type of Student Are You?</h1>
+
+    <form id="quizForm" onsubmit="return validate(event)">
+      <fieldset>
+        <legend>Find Out Your Student Type!</legend>
+
+     
+        <label for="name">Your Name:</label><br>
+        <input type="text" id="name" name="name"><br><br>
+
+  
+        <label for="email">Your Email:</label><br>
+        <input type="email" id="email" name="email"><br><br>
+
+   
+        <p>1. How do you study for exams?</p>
+        <label><input type="radio" name="study" value="early"> I start a week before</label><br>
+        <label><input type="radio" name="study" value="last_minute"> The night before</label><br>
+        <label><input type="radio" name="study" value="never"> I don't study</label><br><br>
+
+   
+        <p>2. Favorite study snack:</p>
+        <label><input type="checkbox" name="snack" value="chips"> Chips</label>
```

Comment:
With checkboxes, we are allowed more than one choice. It this wanted? Then maybe you should clarify the question. Ex: "Favorite study snack(s). Select all that apply."

If you only want one answer, radiobuttons would be better.